webui/src/app/services/data-core-resources.service.ts

```diff
@@ -64,6 +64,15 @@ export class DataCoreResource<T extends EntityBean> implements DefaultResource<T
       .catch(this.handleError);
   }
 
+  /**
+   * get single resource with any reseult
+   */
+  public GetSingleAny = (id: string): Observable<any> => {
```

Comment:
The function name is not meaningful

Reply:
GetSingle return a T type, and in our case it's another type (any)